<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>260</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>455</width>
    <height>260</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>455</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Empty File Generator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="3" rowspan="2">
      <widget class="QPushButton" name="btnGenerate">
       <property name="maximumSize">
        <size>
         <width>999</width>
         <height>999</height>
        </size>
       </property>
       <property name="locale">
        <locale language="Chinese" country="China"/>
       </property>
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblFilePath">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Directory:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="4">
      <widget class="QTextEdit" name="textEditLog">
       <property name="undoRedoEnabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="acceptRichText">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblFileCountTip">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Num of files:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="btnSelect">
       <property name="text">
        <string>Choose</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxSize">
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="maximum">
        <double>10000000000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblFileSizeTip">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;File length:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="cmbUnit">
       <item>
        <property name="text">
         <string>Byte</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>KB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TB</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxCount">
       <property name="decimals">
        <number>0</number>
       </property>
       <property name="maximum">
        <double>1000000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEditFilePath">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
